### ISSUE TITLE
ID selector should be allowed

### ISSUE THREAD

**Issue by yukulele** [CSSLint/csslint#414]
ID selector are useful to stylize unique elements like `#sidebar`, `#footer`, `#logo`

If you don't allow IDs because they are nor reusable, you need also to disallow all unique selector like `html`, `body` and `main`.

but HTML will get ugly:

``` html
<html class="html">
    <body class="body">
        <div id="header" class="header"></div>
        <main class="main">bla</main>
        <div id="footer" class="footer"></div>
    </body>
</html>
```
